src/modelinstance.cpp

```diff
@@ -854,6 +854,7 @@ Status ModelInstance::loadModelImpl(const ModelConfig& config, const DynamicMode
         this->status.setLoading(ModelVersionStatusErrorCode::UNKNOWN);
         return StatusCode::MODEL_NOT_LOADED;
     }
+    SPDLOG_DEBUG("Model loaded from cache: {}", compiledModel->get_property(ov::loaded_from_cache));
```

Comment:
What is the type of returned compiledModel->get_property(ov::loaded_from_cache)? Are we sure all values are convertible to format required by SPDLOG?
To ensure, I would split the call to two, hardcoding the type. Is it bool? If so:
```
bool IsModelLoadedFromCache = compiledModel->get_property(ov::loaded_from_cache);
SPDLOG_DEBUG("Is model loaded from cache: {}", isModelLoadedFromCache);
```

Reply:
does this method return some exception when this property is missing for some plugin different from CPU? I wonder if we should catch such exception